oc-aliases.sh

```diff
@@ -80,29 +87,29 @@ function load_connection() {
+alias tcurl_json='curl -k -H "Authorization: Bearer ${TOKEN}" -H "Content-Type: application/json"'
 
 #COS-SCANNER aliases for running scanner scripts inside docker
-alias cos-notify='oc exec -it --namespace=spectrum-discover $(oc get pods -n=spectrum-discover -l app=spectrum-discover-connmgr -o=jsonpath='{.items[0].metadata.name}') -- bash -c "python3 -m scanners.cloud_scanner.main_notifier"'
-alias cos-replay='oc exec -it --namespace=spectrum-discover $(oc get pods -n=spectrum-discover -l app=spectrum-discover-connmgr -o=jsonpath='{.items[0].metadata.name}') -- bash -c "python3 -m scanners.cloud_scanner.main_replay"'
+alias cos-notify='oc -n ${PROJECT_NAME} exec -c spectrum-discover -it $(oc -n ${PROJECT_NAME} get pods -l role=connmgr -o name) -- bash -c "python3 -m scanners.cloud_scanner.main_notifier"'
+alias cos-replay='oc -n ${PROJECT_NAME} exec -c spectrum-discover -it $(oc -n ${PROJECT_NAME} get pods -l role=connmgr -o name) -- bash -c "python3 -m scanners.cloud_scanner.main_replay"'
 
 # Enable/disable the schedule of the duplicates/mrcapacity scan cronjobs
-alias enabledupjob='gettoken; tcurl_json https://${SD_HOST:?environment variable must be set}/db2whrest/v1/summary_tables/duplicates -X PUT -d'"'"'{"enabled": true}'"'"''
-alias disabledupjob='gettoken; tcurl_json https://${SD_HOST:?environment variable must be set}/db2whrest/v1/summary_tables/duplicates -X PUT -d'"'"'{"enabled": false}'"'"''
-alias enablemrcapjob='gettoken; tcurl_json https://${SD_HOST:?environment variable must be set}/db2whrest/v1/summary_tables/mrcapacity -X PUT -d'"'"'{"enabled": true}'"'"''
-alias disablemrcapjob='gettoken; tcurl_json https://${SD_HOST:?environment variable must be set}/db2whrest/v1/summary_tables/mrcapacity -X PUT -d'"'"'{"enabled": false}'"'"''
-alias rundupjob='gettoken; tcurl_json https://${SD_HOST:?environment variable must be set}/db2whrest/v1/summary_tables/duplicates/start -X PUT'
-alias runmrcapjob='gettoken; tcurl_json https://${SD_HOST:?environment variable must be set}/db2whrest/v1/summary_tables/mrcapacity/start -X PUT'
+alias enabledupjob='gettoken; tcurl_json https://${SD_CONSOLE_ROUTE:?environment variable must be set}/db2whrest/v1/summary_tables/duplicates -X PUT -d'"'"'{"enabled": true}'"'"''
+alias disabledupjob='gettoken; tcurl_json https://${SD_CONSOLE_ROUTE:?environment variable must be set}/db2whrest/v1/summary_tables/duplicates -X PUT -d'"'"'{"enabled": false}'"'"''
+alias enablemrcapjob='gettoken; tcurl_json https://${SD_CONSOLE_ROUTE:?environment variable must be set}/db2whrest/v1/summary_tables/mrcapacity -X PUT -d'"'"'{"enabled": true}'"'"''
+alias disablemrcapjob='gettoken; tcurl_json https://${SD_CONSOLE_ROUTE:?environment variable must be set}/db2whrest/v1/summary_tables/mrcapacity -X PUT -d'"'"'{"enabled": false}'"'"''
+alias rundupjob='gettoken; tcurl_json https://${SD_CONSOLE_ROUTE:?environment variable must be set}/db2whrest/v1/summary_tables/duplicates/start -X PUT'
```

Comment:
Once defined SD_CONSOLE_ROUTE, you can remove the ":?environment variable must be set"